x-pack/plugin/ml/qa/ml-with-security/build.gradle

```diff
@@ -93,7 +93,9 @@ integTestRunner {
     'ml/validate/Test job config that is invalid only because of the job ID',
     'ml/validate_detector/Test invalid detector',
     'ml/delete_forecast/Test delete on _all forecasts not allow no forecasts',
-    'ml/delete_forecast/Test delete forecast on missing forecast'
+    'ml/delete_forecast/Test delete forecast on missing forecast',
+    'ml/results_indices_upgrade/Upgrade results when there is nothing to upgrade',
```

Comment:
Should the names of the yml test change to reflect it's now a generic ML upgrade action?